packages/share_plus/share_plus_platform_interface/lib/method_channel/method_channel_share.dart

```diff
@@ -155,8 +160,13 @@ class MethodChannelShare extends SharePlatform {
   }
 
   /// A wrapper of [MethodChannelShare._getFile] for multiple files.
-  Future<List<XFile>> _getFiles(List<XFile> files) async =>
-      await Future.wait(files.map((entry) => _getFile(entry)));
+  Future<List<XFile>> _getFiles(
+          List<XFile> files, List<String>? fileNameOverrides) async =>
```

Comment:
Add a trailing coma to the last param so it formats nicely

```suggestion
  Future<List<XFile>> _getFiles(
    List<XFile> files, 
    List<String>? fileNameOverrides,
  ) async =>
```